src/compiler/checker.ts

```diff
@@ -10260,7 +10260,8 @@ namespace ts {
             if (getClassExtendsHeritageClauseElement(<ClassDeclaration>node.parent)) {
 
                 if (containsSuperCall(node.body)) {
-                    // The first statement in the body of a constructor must be a super call if both of the following are true:
+                    // The first statement in the body of a constructor must be a super call(or prologue directives followed by a super call)

```

Comment:
Maybe just turn this into

```TypeScript
// The first statement in the body of a constructor (excluding prologue directives) must be a super call.
```

If you prefer your original wording, just split the comment into two linese.